_includes/challenge_card.html

```diff
@@ -45,6 +45,28 @@
             </li>
         {% endif %}
 
+        {% if challenge.file_path %}
```

Comment:
Was this supposed to be `challenge.hints` ?

Reply:
Yeap, thanks